app/src/main/java/de/danoeh/antennapod/adapter/DownloadStatisticsListAdapter.java

```diff
@@ -0,0 +1,88 @@
+package de.danoeh.antennapod.adapter;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.request.RequestOptions;
+
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.core.glide.ApGlideSettings;
+import de.danoeh.antennapod.core.storage.DBReader;
+import de.danoeh.antennapod.core.util.Converter;
+
+/**
+ * Adapter for the download statistics list
+ */
+public class DownloadStatisticsListAdapter extends StatisticsListAdapter {
+
+    public DownloadStatisticsListAdapter(Context context) {
+        super(context);
+    }
+
+    @Override
+    public int getItemCount() {
+        return statisticsData.feedTime.size() + 1;
+    }
+
+    public DBReader.StatisticsItem getItem(int position) {
+        if (position == 0) {
+            return null;
+        }
+        return statisticsData.feedTime.get(position - 1);
+    }
```

Comment:
The above two methods can be removed from the child classes as they already are in the parent class.